src/runtime.ts

```diff
@@ -14,32 +12,54 @@ const runtime: DefinitionsCall = {
               type: "Text",
             },
           ],
-          type: "Option<RawDidLinkedInfo<AccountId32, AccountId32, AccountId32, Balance, Key, BlockNumber>>"
+          type: "Option<RawDidLinkedInfo<AccountId32, AccountId32, AccountId32, Balance, Key, BlockNumber>>",
         },
         query_did_by_account_id: {
           description: "Return the information relative to the DID to which the provided account is linked, if any.",
           params: [
             {
               name: "account",
-              type: "AccountId32"
-            }
+              type: "AccountId32",
+            },
           ],
-          type: "Option<RawDidLinkedInfo<AccountId32, AccountId32, AccountId32, Balance, Key, BlockNumber>>"
+          type: "Option<RawDidLinkedInfo<AccountId32, AccountId32, AccountId32, Balance, Key, BlockNumber>>",
         },
         query_did: {
           description: "Return the information relative to the owner of the provided DID, if present.",
           params: [
             {
               name: "did",
-              type: "AccountId32"
-            }
+              type: "AccountId32",
+            },
           ],
-          type: "Option<RawDidLinkedInfo<AccountId32, AccountId32, AccountId32, Balance, Key, BlockNumber>>"
-        }
+          type: "Option<RawDidLinkedInfo<AccountId32, AccountId32, AccountId32, Balance, Key, BlockNumber>>",
+        },
+      },
+      version: 1,
+    },
+  ],
+  ParachainStakingApi: [
+    {
+      methods: {
+        get_staking_rates: {
+          description: "Calculates the current staking and reward rates for collators and delegators",
```

Comment:
```suggestion
          description: "Calculate the current staking and reward rates for collators and delegators.",
```

Reply:
It could also be the other way around, where the DID descriptions use the 3rd person.